core/src/main/java/io/questdb/cutlass/http/HttpFullFatServerConfiguration.java

```diff
@@ -28,12 +28,72 @@
 import io.questdb.cutlass.http.processors.LineHttpProcessorConfiguration;
 import io.questdb.cutlass.http.processors.StaticContentProcessorConfiguration;
 import io.questdb.std.ConcurrentCacheConfiguration;
+import io.questdb.std.ObjList;
 
 public interface HttpFullFatServerConfiguration extends HttpServerConfiguration {
     String DEFAULT_PROCESSOR_URL = "*";
 
     ConcurrentCacheConfiguration getConcurrentCacheConfiguration();
 
+    default ObjList<String> getContextPathDefault() {
+        return new ObjList<>() {{
+            add(DEFAULT_PROCESSOR_URL);
+        }};
+    }
+
+    default ObjList<String> getContextPathExec() {
```

Comment:
i think these could be simplified as
```
return new ObjList<>("/exec");
```

Reply:
done in https://github.com/questdb/questdb/pull/5297/commits/ff1596202714534986570d5b1e9c77062120971f